string config_file_path
string vocab_file_path
string camera_topic
---
